src/main/java/com/softserve/app/service/userService/UserServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.softserve.app.service.userService;
+
+import com.softserve.app.models.User;
+import com.softserve.app.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public abstract class UserServiceImpl implements UserService {
+
+    @Autowired
+    private final UserRepository userRepository;
+
+    @Autowired
+    public UserServiceImpl(UserRepository userRepository){
+        this.userRepository = userRepository;
+    }
+
+    public List<User> findAll() {
+        return  userRepository.findAll();
+    }
+
+    public User findById(Long id) {
+        if (userRepository.findById(id).isPresent()) {
```

Comment:
userRepository.findById(id).orElseThrow( () -> new CustomException("user not found", 500))